requirements.txt

```diff
@@ -1,2 +1,3 @@
 jupyter-book==0.12.3
 pyppeteer
+pybtex-apa-style
```

Comment:
It might be a good idea to use `pybtex-apa-style>=1.3`.

Reply:
Thanks for the review. At the moment we use `btex-apa-style` without any version restrictions 
for the other SOPs where we have already changes the citation style. 

What is the reasoning behind restricting it to >=1.3?

Reply:
To inform everyone that you successfully ran it with `pybtex-apa-style-1.3`. Someone trying to replicate what you did might find errors. By informing the version that you used, we restrict the possible sources for the error and make it easier to reproduce what you did. The 1.2 might work as well, but since you didn't test it, better say 1.3 and hopefully the following ones. A note, I checked the actions log to determine that 1.3 was used.

Some people would pin the exact version, like done for the `jupyter-book==0.12.3`. That holds back any improvements from newer versions but guarantees that one year down the road the same book will compile again in case newer versions of the `jupyter-book` changes the syntax. One alternative would be using `pybtex-apastyle~=1.3.0`, so it would hold on 1.3 while allowing for small fixes, but not 1.4.

Reply:
Thanks a lot @castelao for taking the time to explain this in detail to me. 

Indeed we had to pin to jupyter-book==0.12.3 due to this bug: https://github.com/OceanGlidersCommunity/Oxygen_SOP/issues/234
https://github.com/executablebooks/jupyter-book/issues/1762

Reply:
FYI @callumrollo @jbusecke